psalm.xml.dist

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0"?>
+<psalm
+        autoloader="bin/.phpunit/phpunit-8.5-0/vendor/autoload.php"
```

Comment:
Not sure why but I had to change this to `bin/.phpunit/phpunit-8.3-0/vendor/autoload.php` on my local, even with fresh install